config/initializers/twitter_client.rb

```diff
@@ -0,0 +1,7 @@
+require 'twitter'
+::Client = Twitter::REST::Client.new do |config|
+  config.consumer_key        = ENV["TWITTER_CONSUMER_KEY"]
```

Comment:
OK, so do we need new keys for these and where should I look for them? Does it need to be the @publiclab twitter or could it be another? Thanks, just checking!!

Reply:
It can be of any account...